test/parallel/test-whatwg-webstreams-encoding.js

```diff
@@ -0,0 +1,102 @@
+// Flags: --expose-internals --no-warnings
```

Comment:
I think we can avoid the expose-internals if the classes are imported from the public module